x-pack/plugins/apm/common/rules/apm_rule_types.ts

```diff
@@ -294,6 +298,35 @@ export type AnomalyAlertSeverityType = ValuesType<
   typeof ANOMALY_ALERT_SEVERITY_TYPES
 >['type'];
 
+export function getApmMlDetectorLabel(type: ApmMlDetectorType) {
+  switch (type) {
+    case ApmMlDetectorType.txLatency:
+      return i18n.translate('xpack.apm.alerts.anomalyDetector.latencyLabel', {
+        defaultMessage: 'latency',
+      });
+    case ApmMlDetectorType.txThroughput:
+      return i18n.translate(
+        'xpack.apm.alerts.anomalyDetector.throughputLabel',
+        {
+          defaultMessage: 'throughput',
+        }
+      );
+    case ApmMlDetectorType.txFailureRate:
+      return i18n.translate(
+        'xpack.apm.alerts.anomalyDetector.failedTransactionRateLabel',
+        {
+          defaultMessage: 'failed transaction rate',
+        }
+      );
+  }
+}
```

Comment:
This is duplicated in x-pack/plugins/apm/public/components/alerting/rule_types/anomaly_rule_type/get_apm_ml_detector_label.ts ?

Reply:
Leftover, deleted